rabbitmq/metadata.csv

```diff
@@ -78,223 +78,226 @@ rabbitmq.overview.messages.publish_in.count,gauge,,message,,Count of messages pu
+rabbitmq.erlang.uptime_seconds,gauge,,second,,[OpenMetricsV2] Node uptime,0,rabbitmq,uptime seconds,
+rabbitmq.erlang.vm.allocators,gauge,,,,[OpenMetricsV2] Allocated (carriers_size) and used (blocks_size) memory for the different allocators in the VM. See erts_alloc(3).,0,rabbitmq,erlang vm allocators,
+rabbitmq.erlang.vm.atom_count,gauge,,,,[OpenMetricsV2] The number of atom currently existing at the local node.,0,rabbitmq,atom count,
+rabbitmq.erlang.vm.atom_limit,gauge,,,,[OpenMetricsV2] The maximum number of simultaneously existing atom at the local node.,0,rabbitmq,atom limit,
+rabbitmq.erlang.vm.dirty_cpu_schedulers_online,gauge,,,,[OpenMetricsV2] The number of dirty CPU scheduler threads online.,0,rabbitmq,dirty cpu schedulers online,
+rabbitmq.erlang.vm.dirty_cpu_schedulers,gauge,,,,[OpenMetricsV2] The number of scheduler dirty CPU scheduler threads used by the emulator.,0,rabbitmq,dirty cpu schedulers,
+rabbitmq.erlang.vm.dirty_io_schedulers,gauge,,,,[OpenMetricsV2] The number of scheduler dirty I/O scheduler threads used by the emulator.,0,rabbitmq,dirty io schedulers,
+rabbitmq.erlang.vm.dist.node_queue_size_bytes,gauge,,byte,,[OpenMetricsV2] The number of bytes in the output distribution queue. This queue sits between the Erlang code and the port driver.,0,rabbitmq,node queue size bytes,
+rabbitmq.erlang.vm.dist.node_state,gauge,,,,"[OpenMetricsV2] The current state of the distribution link. The state is represented as a numerical value where `pending=1', `up_pending=2' and `up=3'.",0,rabbitmq,node state,
+rabbitmq.erlang.vm.dist.port_input_bytes,gauge,,byte,,[OpenMetricsV2] The total number of bytes read from the port.,0,rabbitmq,port input bytes,
+rabbitmq.erlang.vm.dist.port_memory_bytes,gauge,,byte,,[OpenMetricsV2] The total number of bytes allocated for this port by the runtime system. The port itself can have allocated memory that is not included.,0,rabbitmq,port memory bytes,
+rabbitmq.erlang.vm.dist.port_output_bytes,gauge,,byte,,[OpenMetricsV2] The total number of bytes written to the port.,0,rabbitmq,port output bytes,
+rabbitmq.erlang.vm.dist.port_queue.size_bytes,gauge,,byte,,[OpenMetricsV2] The total number of bytes queued by the port using the ERTS driver queue implementation.,0,rabbitmq,size bytes,
+rabbitmq.erlang.vm.dist.proc.heap_size_words,gauge,,,,"[OpenMetricsV2] The size in words of the youngest heap generation of the process. This generation includes the process stack. This information is highly implementation-dependent, and can change if the implementation changes.",0,rabbitmq,heap size words,
+rabbitmq.erlang.vm.dist.proc.memory_bytes,gauge,,byte,,"[OpenMetricsV2] The size in bytes of the process. This includes call stack, heap, and internal structures.",0,rabbitmq,memory bytes,
+rabbitmq.erlang.vm.dist.proc.message_queue_len,gauge,,,,[OpenMetricsV2] The number of messages currently in the message queue of the process.,0,rabbitmq,message queue len,
```

Comment:
```suggestion
rabbitmq.erlang.vm.dist.proc.message_queue_len,gauge,,message,,[OpenMetricsV2] The number of messages currently in the message queue of the process.,0,rabbitmq,message queue len,
```
https://docs.datadoghq.com/metrics/units/#overview

Reply:
I wasn't sure if those are the same messages, because the DD ones are in a group called "network". I thought maybe there are more lower-level messages that take place in the networking stack. But I defer to you on this.

Reply:
Ah that makes sense. I think these are queue messages, so let's leave it as-is